build.gradle.kts

```diff
@@ -57,11 +64,10 @@ intellij {
     pluginName.set(properties("pluginName"))
     version.set(properties("platformVersion"))
     type.set(properties("platformType"))
-    //localPath.set("/Applications/IntelliJ IDEA.app")
+    // localPath.set("/Applications/IntelliJ IDEA.app")
```

Comment:
This and all changes below (except one with null-safe chain) are the outcome of linting the code with recent ktlint.